controllers/ebpf/agent-metrics-test.go

```diff
@@ -1,56 +0,0 @@
-package ebpf
```

Comment:
any reason for deleting this unit-test ?

Reply:
Yes I explained it in the commit message :)
[dc257fb](https://github.com/netobserv/network-observability-operator/pull/674/commits/dc257fb3931d98487f120e44d1e83f916ef660cc)